src/test/java/seedu/guest/testutil/TypicalPersons.java

```diff
@@ -66,15 +66,15 @@ private TypicalPersons() {} // prevents instantiation
     /**
      * Returns an {@code AddressBook} with all the typical persons.
```

Comment:
Should this be guests? Also, should the TypicalPersons be renamed to TypicalGuests?